devel/darwinbuild-legacy/Portfile

```diff
@@ -1,53 +1,55 @@
 PortSystem      1.0
+PortGroup       github 1.0
 
 name            darwinbuild-legacy
-version         0.8.0
-revision	655
 categories      devel
 platforms       darwin
+license         BSD
 maintainers     wms
 description     Darwinbuild is a set of tools that facilitate building the sources \
-		released by Apple.
+                released by Apple.
 
 long_description \
-		The Darwin Build Scripts are a collection of tools that assist compilation of \
-		the many projects contained in Darwin, the open source base of Apple's \
-		Mac OS X operating system.  Apple publishes the sources of these projects \
-		in an archive format (.tar.gz).  An archive is published for each project \
-		version on Apple's site. These tools will provide the proper build environment as well as help to \
-		resolve any necessary dependencies prior to building.
-
-homepage        http://darwinbuild.macosforge.org/
```

Comment:
Homepage is still needed. It seems to be at https://macosforge.github.io/darwinbuild/.
(But don't worry adding it now, I'm in the middle of committing.)